src/components/typography/typography.css

```diff
@@ -75,3 +75,57 @@ p,
 .body-sm {
 	font-size: 8px;
 }
+
+.text_comp__wrapper {
+	align-content: stretch;
+}
+
+.text__wrapper {
+	margin-bottom: 5px;
+	display: flex;
+	align-items: center;
+}
+
+.text__info {
+	margin-left: auto;
+}
+
+.text__xxxl {
+	font-size: 2em;
+}
+
+.text__xxl {
+	font-size: 1.5em;
+}
+
+.text__xl {
+	font-size: 1.25em;
+}
+
+.text__lg {
+	font-size: 1.17em;
+}
+
+.text__md {
+	font-size: 1em;
+}
+
+.text__sm {
+	font-size: 0.83em;
```

Comment:
for size less than 1 em you can use pixels

Reply:
Here: I want to use em to size relative to parent font size.